tokio/tests/sync_broadcast.rs

```diff
@@ -457,6 +457,20 @@ fn lagging_receiver_recovers_after_wrap_open() {
     assert_empty!(rx);
 }
 
+#[test]
```

Comment:
Do we need both tests? This looks to be the same as our doc test.

Reply:
No, thanks. Forgot that tokio tests the examples in the docs.